### PR TITLE
Update version in example file to known good version of openshift dep…

### PR DESCRIPTION
**Pull request by dmc5179**
…loyment scripts

### CODE REVIEW

File: vars/aws.example.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ cluster_name: openshift
 
 # The OpenShift version to install
 # IMPORTANT: Make sure this value is quoted, otherwise it gets read as 3.1 instead of 3.10
-openshift_version: "3.10"
+openshift_version: "3.11.43"
```
Comment (jaredhocutt) [inline vars/aws.example.yml]: The value for `openshift_version` must be `3.xx` and doesn't support the format `3.xx.xx`.

To set a z-stream release, there is a separate variable named `openshift_version_minor`.

In this example, you'd set the following:

```yaml
openshift_version: "3.11"
openshift_version_minor: "3.11.43"
```

**Review by jaredhocutt — CHANGES_REQUESTED** [review]
Thanks for the PR!

I forgot to update the example to `3.11`. Check the comment on the code for the small change to make. Push an update and I'll get it merged.

**Review by jaredhocutt — APPROVED** [review]
LGTM

Thanks!

**Comment by dmc5179** [comment]
Haha that was fast. I'm familiar with git but new to pull requests. I take it from the sequence of events that by pushing a new change to my fork that it was added into the same pull request. I'll have to keep that in mind for the future. Thank you.

**Pull request merged**